src/Activity.js

```diff
@@ -1,24 +1,17 @@
-class Activity {
+import HealthMonitor from './Health-monitor'
+
+class Activity extends HealthMonitor {
   constructor(activityData) {
-    this.activityData = activityData
+    super(activityData)
   }
 
   getMilesFromStepsByDate(id, date, userRepo) {
-    const userStepsByDate = this.activityData.find(data => {
-      return id === data.userID && date === data.date
-    });
-    return parseFloat(((userStepsByDate.numSteps * userRepo.strideLength) / 5280).toFixed(1));
-  }
-
-  getActiveMinutesByDate(id, date) {
-    const userActivityByDate = this.activityData.find(data => {
-      return id === data.userID && date === data.date
-    });
-    return userActivityByDate.minutesActive;
+    const userStepsByDate = this.calculateDaily(id, date, "numSteps");
```

Comment:
Great job DRYing this up by \ calling the calculateDaily() method in here and inside of the accomplishStepGoal() method to get rid of code that was nearly identical.